### PR TITLE
Fixing the problem of  Dead link in documentation.

### PR DESCRIPTION
**Pull request by alok1304**
#540 fix. Now I fix the problem of Dead link in documentation.
Screenshot 2024-11-05 115146

### CODE REVIEW

File: docs/source/matchcode/matchcode-pipeline.rst [hunk 1]
```diff
@@ -4,11 +4,12 @@ Code Matching
 The aim of this tutorial is to show how to use the MatchCode.io API to perform
 code matching on an archive of files.
 
+
 .. note::
     This tutorial assumes that you have a working installation of PurlDB. If you
     don't, please refer to the `installation
-    <../purldb/overview.html#installation>`_ page.
-
+    <https://aboutcode.readthedocs.io/projects/PURLdb/en/latest/how-to-guides/installation.html>`_
```
Comment (JonoYang) [inline docs/source/matchcode/matchcode-pipeline.rst:11]: This should not be an absolute link to the readthedocs page. We want to use cross-references (https://www.sphinx-doc.org/en/master/usage/referencing.html)

File: package-lock.json
Comment (JonoYang) [inline package-lock.json:1]: This file should be removed.

**Review by JonoYang — CHANGES_REQUESTED** [review]
@alok1304 Thanks for the PR, I left some comments.

**Review by alok1304 — COMMENTED** [review]
finally i do it.

**Pull request closed without merging**